### DISCUSSION TITLE
Limit whitespace at the end of generated file

### DISCUSSION THREAD

**Discussion by slowy07** [lowlighter/metrics#493] [Q&A] [answered]
hello!,
i have question about metrics generated to svg, and  make space to much like this  after displaying
![mydiscuss](https://user-images.githubusercontent.com/40540262/130320476-8997176b-c0a2-4dbc-ad10-1e2f6c89db7a.png)
how can i set more close and don't make space to much for this ??, thanks !

**Comment by lowlighter** [comment, marked as answer]
You can try to tweak this by changing the following value:
https://github.com/lowlighter/metrics/blob/19405ee4dc2d5607acfe94d175ed0173429095d1/source/plugins/core/metadata.yml#L186-L192

Maybe try to set a lower percentage, like 7-8%

Related to:
* #321

**Reply by slowy07** [reply]
thanks sir @lowlighter , sorry, i'm to much asking 😂 

thanks sir, have a good day 👍🏻 💯
